components/docs-chef-io/content/automate/restore.md

```diff
@@ -193,6 +193,11 @@ Set the log level to `debug` before re-running a failed restore to output debug
 chef-automate debug set-log-level deployment-service debug
 ```
 
+If you are restoring to a machine with less memory than the original system, you my need to add the following to your config patch (adjusted for your appropriate memory settings):
```

Comment:
```suggestion
If you are restoring to a machine with less memory than the original system, you may need to add the following to your config patch (adjusted for your appropriate memory settings):
```
"my" in the original change should be "may"

Reply:
Would it be worth putting this in the Prerequisites section on the Restore page, rather than in the Troubleshooting section?

Reply:
I'm happy to have it recorded anywhere, as long as it's documented.

Reply:
@mattray This shouldn't block merge; I'm happy getting this in as-is and then moving the text later.